src/main/java/hwr/oop/group4/todo/ui/TaskUi.java

```diff
@@ -0,0 +1,234 @@
+package hwr.oop.group4.todo.ui;
+
+import hwr.oop.group4.todo.commons.exceptions.TodoRuntimeException;
+import hwr.oop.group4.todo.core.*;
+import hwr.oop.group4.todo.ui.controller.ConsoleController;
+import hwr.oop.group4.todo.ui.controller.ConsoleHelper;
+import hwr.oop.group4.todo.ui.controller.command.Command;
+import hwr.oop.group4.todo.ui.controller.command.CommandArgument;
+import hwr.oop.group4.todo.ui.controller.menu.Entry;
+import hwr.oop.group4.todo.ui.controller.menu.EntryArgument;
+import hwr.oop.group4.todo.ui.controller.menu.Menu;
+import hwr.oop.group4.todo.ui.controller.tables.ColumnConfig;
+import hwr.oop.group4.todo.ui.controller.tables.Table;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class TaskUi {
```

Comment:
This class is a bit long. Do you have any ideas how we could shorten it?

Reply:
Maybe we can extract actions like create or edit into extra classes, e.g. CreateTaskUi, EditTaskUi, ...